dev/io.openliberty.grpc.1.0.internal.monitor/bnd.bnd

```diff
@@ -30,7 +30,7 @@ Import-Package: \
   com.ibm.websphere.pmi;version="[1.1,2)", \
```

Comment:
Update copyright date range to include 2023

Reply:
Done